Artifacts/MsSql/Data/Security/1030-DataStandard3.3a-ResourceClaimMetadata.sql

```diff
@@ -0,0 +1,33 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+DECLARE @applicationId INT;
+DECLARE @systemDescriptorsResourceClaimId INT;
+DECLARE @relationshipBasedDataResourceClaimId INT;
+
+SELECT @applicationId = (SELECT applicationid FROM  dbo.Applications  WHERE  ApplicationName  = 'Ed-Fi ODS API');
+SELECT @systemDescriptorsResourceClaimId = (SELECT ResourceClaimId FROM dbo.ResourceClaims WHERE ResourceName = 'systemDescriptors' AND Application_ApplicationId = @applicationId);
+SELECT @relationshipBasedDataResourceClaimId = (SELECT ResourceClaimId FROM dbo.ResourceClaims WHERE ResourceName = 'relationshipBasedData' AND Application_ApplicationId = @applicationId);
+
+/* new descriptors */
+IF (NOT EXISTS (SELECT ResourceClaimId FROM dbo.ResourceClaims WHERE ResourceName = 'AncestryEthnicOriginDescriptor' AND Application_ApplicationId = @applicationId))
+BEGIN
+	INSERT dbo.ResourceClaims (DisplayName, ResourceName, ClaimName, ParentResourceClaimId, Application_ApplicationId)
+	VALUES (N'AncestryEthnicOriginDescriptor', N'AncestryEthnicOriginDescriptor', N'http://ed-fi.org/ods/identity/claims/AncestryEthnicOriginDescriptor', @systemDescriptorsResourceClaimId, @applicationId);
```

Comment:
I left a similar comment on the Postgres version of this script, but you need to follow the conventions for the claim name URIs... which is to camel-case the resource name, as follows:
`http://ed-fi.org/ods/identity/claims/ancestryEthnicOriginDescriptor`
... not ... `http://ed-fi.org/ods/identity/claims/AncestryEthnicOriginDescriptor`.

Apply this convention to all resource claims you're creating.

Reply:
I fixed it . Thanks